### PR TITLE
Make mime-types and rest-client dependency constraints platform-specific

### PR DESCRIPTION
**Pull request by sferik**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,10 @@
 source 'https://rubygems.org'
 
+platforms :ruby_19 do
```
Comment (alexmamonchik) [inline Gemfile]: this is exactly what I looked for!

**Comment by sferik** [comment]
Tests are passing, so this should be good to merge now. :shipit:

**Comment by sferik** [comment]
If you would be so kind as to do a gem release after merging this, all of us at Instacart would greatly appreciate it! 💎

**Comment by alexmamonchik** [comment]
Dear @mindeavor, please, do gem release

**Comment by mindeavor** [comment]
Published a37f724e5c8a72482b7294bf1a6b0aa6a1eed50a

**Pull request merged**